Simplified/Reader2/AdobeDRM/AdobeDRMContentProtection.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  AdobeDRMContentProtection.swift
+//  Simplified
+//
+//  Created by Vladimir Fedorov on 20.01.2021.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+import R2Shared
+
+#if FEATURE_DRM_CONNECTOR
+
+class AdobeDRMContentProtection: ContentProtection {
+  func open(asset: PublicationAsset, fetcher: Fetcher, credentials: String?, allowUserInteraction: Bool, sender: Any?, completion: @escaping (CancellableResult<ProtectedAsset?, Publication.OpeningError>) -> Void) {
+    
+    // Publication asset must be a FileAsset, as we are opening a file
+    // If not, we can't open and use Adobe DRM
+    guard let fileAsset = asset as? FileAsset else {
+      completion(.failure(.unavailable(nil)))
```

Comment:
should we add a `Log.error` line to this?

Reply:
It's a good question if it can be anything other than file... FileAsset is the only class implementing PublicationAsset protocol at the moment.

Reply:
in that case it might be good to add one for future proofing

Reply:
added